jrodriguezdiazz/jrodriguezdiazz-portfolio
# add-book-engine-project

## Description

Add the first project to the portfolio


Add the components:
- _ProjectImage_: Render an image related to the project

- _ProjectDescription_: Renders data related to the project (Name, GitHub and Website links and technology stack used).
  - **Note**: This component is hidden on mobile devices



---

## Motivation and Context



This change is required because it was necessary to develop the necessary components to show the projects developed by me.



---

## How Has This Been Tested?





1. Go to the next page: https://jrodriguezdiazz-portfolio-kb9c29vc3-jrodriguezdiazz.vercel.app/projects#projects
2. Check that a card is rendered that details the information of the `Book Search Engine` project

---

## Screenshots (if appropriate):

| Desktop                                                                                                         | Mobile                                                                                                          |
|-----------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
| ![image](https://user-images.githubusercontent.com/24766287/158901082-505f60cf-2ce1-47e9-9c14-3d943c2a3f5e.png) | ![image](https://user-images.githubusercontent.com/24766287/158901202-835c0817-ae37-4e4d-bdea-49ef362efa6e.png) |
|

---

## Types of Changes




---

## Checklist:





---

# Other Notes
